sirepo_bluesky/sirepo_flyer.py

```diff
@@ -232,12 +233,16 @@ def kickoff(self):
             c1.data['report'] = 'watchpointReport{}'.format(watch['id'])
             self._copies.append(c1)
 
+        start_time = ttime.monotonic()
```

Comment:
The `start_time` may not be needed here. See my comment on the `_run()` method's signature.